src/RepoToolset/GenerateResxSource.targets

```diff
@@ -48,4 +54,5 @@
       </EmbeddedResourceSG>
     </ItemGroup>
   </Target>
+
 </Project>
```

Comment:
Extra empty line

Reply:
The convention in all the shipping props/targets has been to have an extra newline after the Project open tag and before the Project close tag.